ktor-server/ktor-server-core/nonJvm/src/io/ktor/server/engine/EnvironmentUtils.nonJvm.kt

```
@@ -18,10 +15,7 @@ internal actual fun ApplicationEngine.Configuration.configureSSLConnectors(
     sslTrustStorePassword: String?,
     sslEnabledProtocols: List<String>?
 ) {
-    error("SSL is not supported in native")
+    error("SSL is not supported on non JVM platforms")
 }
 
 internal actual fun ApplicationEnvironmentBuilder.configurePlatformProperties(args: Array<String>) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)